hack/verify-govet.sh

```diff
@@ -18,4 +18,4 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
-go vet $(go list ./... | grep -v vendor)
+go vet ./...
```

Comment:
go vet ignores vendor now

Reply:
in slightly more detail: `golint` doesn't, but `go list ./...` and `go vet ./...` [since go 1.9](https://github.com/golang/go/commit/fa1d54c2edad607866445577fe4949fbe55166e1), as of today we're on go1.11